docs/building.rst

```diff
@@ -10,19 +10,18 @@ Building ctags
 Build system add possibility to change program name
 ---------------------------------------------------------------------
 
-As on some systems (e.g. BSD) there is a 'ctags' program in the base
-system it's somewhat inconvenient to have the same name for universal-ctags
-During ``configure`` you can now change the output executable name.
+As on some systems (e.g. BSD), there is alreayd a 'ctags' program in the base
```

Comment:
Or maybe: On some systems, like BSD, there is already a 'ctags' program in the base